test/specs/controller.scatter.test.js

```diff
@@ -0,0 +1,115 @@
+describe('Chart.controllers.scatter', function() {
+
+	it('should not draw a line if the options showLines is not set', function() {
+		var chart = window.acquireChart({
+			type: 'scatter',
+			data: {
+				datasets: [{
+					data: [
```

Comment:
We don't need so much data, maybe one is enough `data: [{x: 10, y: 15}]` since we are not verifying how the line is drawing but only if the method is called. That would make all tests shorter.